pkg/sippyserver/dbloader.go

```diff
@@ -383,13 +385,23 @@ func appendJobIssuesFromVariants(jobCache map[string]*models.ProwJob, jobIssues
 			}
 
 			for key, issues := range variantsIssuesMap {
-				if variantsSetMap[variantsKey].IsSuperset(variantsSetMap[key]) {
-					jobSearchStrings := fmt.Sprintf("job=%s=all", job.Name)
-					if _, ok := jobIssues[jobSearchStrings]; !ok {
-						jobIssues[jobSearchStrings] = []jira.Issue{}
+				if !variantsSetMap[variantsKey].IsSuperset(variantsSetMap[key]) {
+					continue
+				}
+				candidates := []jira.Issue{}
+				for _, issue := range issues {
+					for _, version := range issue.Fields.AffectsVersions {
+						if job.Release == version.Name {
```

Comment:
AffectsVersions can be 4.12, 4.12.z, 4.12.1

We should be covered with something like:

if job.Release == version.Name || strings.HasPrefix(version.Name, job.Release + ".") 

Also trying to prevent 4.1 matching 4.12.